test.py

```diff
@@ -1,12 +1,37 @@
-from number.FloatNumber import FloatNumber
+# ckrawczyk
+# mrybarczyk
+
 from number.NumberFabric import NumberFabric
 from number.NumberType import NumberType
+from matrix.MyMatrix import MyMatrix
+import numpy
+import random
 
-print("test")
-
+globalMax = 65536
 fabric = NumberFabric()
 numberType = NumberType.FLOAT
-number = fabric.createNumberFromType(numberType, 5, 1)
-number.print()
-number.add(FloatNumber(10, 2))
-print(number.value)
+
+def createMatrix(size, numberType):
```

Comment:
Ogólnie całe to generowanie bym wydzielił do oddzielnej klasy, bo będizemy mogli to jakoś przetestować fajnie i ustawić na sztywno dane wartości

Reply:
Jako metodę MyMatrix czy jak?